perun/profile/helpers.py

```diff
@@ -609,3 +610,51 @@ def is_compatible_with_profile(self, profile: profiles.Profile) -> bool:
         "checksum",
         "source",
     ]
+
+
+@dataclass
+class ProfileStat:
+    ALLOWED_ORDERING: ClassVar[dict[str, bool]] = {
+        "higher_is_better": True,
```

Comment:
I would consider simplifying to `better` with type `Literal["higher", "lower"]`. I could imagine, that there might be case, when there is something different (maybe equal?).